tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Deduplication/DeduplicationTests.cs

```diff
@@ -36,8 +36,10 @@ public DeduplicationTests(ITestOutputHelper output)
     [InlineData(true)]
     public async Task SubmitsTraces(bool deduplicationEnabled)
     {
-        SetEnvironmentVariable("DD_IAST_ENABLED", "true");
+        SetEnvironmentVariable("DD_TRACE_DEBUG", "1");
+        SetEnvironmentVariable("DD_IAST_ENABLED", "1");
         SetEnvironmentVariable("DD_IAST_DEDUPLICATION_ENABLED", deduplicationEnabled.ToString());
+        SetEnvironmentVariable("DD_TRACE_LOG_DIRECTORY", Path.Combine(EnvironmentHelper.LogDirectory, "DeduplicationLogs"));
```

Comment:
I'm guessing this was just for debugging. Probably not necessary now?

Reply:
Done!